guard/src/commands/migrate.rs

```diff
@@ -34,7 +34,7 @@ impl Command for Migrate {
         MIGRATE
     }
 
-    fn command(&self) -> App<'static, 'static> {
+    fn command(&self) -> App<'static> {
```

Comment:
Is this change a result of upgrading clap or are we changing something? Also, `ArgMatches<'_>`?